vast/vast.cpp

```diff
@@ -33,6 +34,23 @@
 using namespace vast;
 using namespace vast::system;
 
+std::string format_error(caf::error err) {
+  std::ostringstream oss;
+  auto category = err.category();
+  oss << "Error: ";
+  if (category == caf::atom("vast"))
```

Comment:
Personally, I find if-else trees harder to read than switch-case. Did you thought of something like this?

```c++
switch (atom_uint(err.category())) {
  case atom_uint("vast"):
  // ...
}
```